onnx/version_converter/adapters/axis_attribute_to_input.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) ONNX Project Contributors
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
```

Comment:
That would be a great thing to have. @gramalingam @liqunfu what do you think?

Reply:
What can we avoid if we support multiple axes? The reason for supporting a single axis, as I remember it, is that the exporter/user can use a series of DFT ops (to simulate multiple axes). But I am open to having a multiple-axes version of the op if there is a good reason. (But we should also consider the code-freeze for this release.)

Reply:
I think @xadupre meant the creation of this adapter can be avoided because we can reuse the axes_xxx adapters. I think the current single axis makes sense for the usage required and considering the release timeline.